mail_factory/tests/test_views.py

```diff
@@ -23,6 +24,27 @@ def test_get_context_data(self):
         self.assertEqual(len(data), len(factory._registry))
 
 
+class TemplateTest(TestCase):
+    """
+    Simple test case to get a view and check the template doesn't
+    raise any TemplateError.
+
+    """
+    def setUp(self):
+        super(TemplateTest, self).setUp()
+
+        credentials = {
+            'username': 'admin',
+            'password': 'admin',
+        }
+        User.objects.create_superuser(email='admin@example.com', **credentials)
+        self.client.login(**credentials)
+
+    def test_get_mail_factory_list(self):
+        response = self.client.get(reverse('mail_factory_list'))
```

Comment:
You are not checking that the DeprecationWarning disappeared which was the original bug you were trying to solve are you?

Reply:
I add 80792e264263020425d4a622dd1b07ced40e3a27. Is it OK for you @Natim?
(if yes, I'll squash commits)

Reply:
Arf. OK, need to dig a little bit more.